### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by ghost** [The-Lively-Developers-Community/Lively-Game#6] [Ideas]
The game itself is nice as a concept but it has quite some problems.
- The game looks too generic (skybox,gui,floor etc)
- When you fall out of the floor it doesn't respawn
- No pause system
Overall the game and the concept is pretty good and can make a good game but these are a few problem with the game.

**Comment by realguystuff** [comment]
if you fall out, you can press R to restart the level (and the stopwatch)

**Comment by realguystuff** [comment]
others are planned for alpha (which is the current phase rn, prealpha done)

**Comment by ghost** [comment]
ok

**Comment by KendallDoesCoding** [comment]
Transferred issue to a discussion @shourdev @realguystuff .
